src/Ninja/G1CP/Content/Testsuite/getter.d

```diff
@@ -0,0 +1,120 @@
+/*
+ * Check if item exists and return its symbol index
+ */
+func int G1CP_Testsuite_GetItemId(var string name) {
+    // Check if item instance exists
+    if (!G1CP_IsItemInst(name)) {
+        var string msg; msg = ConcatStrings("Item '", name);
+        msg = ConcatStrings(msg, "' not found");
+        G1CP_TestsuiteErrorDetail(msg);
+        G1CP_TestsuiteStatusPassed = FALSE;
+    };
+
+    // Return the item's symbol index
+    return MEM_GetSymbolIndex(name);
+};
+
+/*
+ * Check if integer variable exists and return its symbol pointer
+ */
+func int G1CP_Testsuite_GetIntVarPtr(var string name, var int arrIdx) {
+    // Check if variable exists
+    if (!G1CP_IsIntVar(name, arrIdx)) {
+        var string msg; msg = ConcatStrings("Integer variable '", name);
+        msg = ConcatStrings(msg, "' not found/valid");
+        G1CP_TestsuiteErrorDetail(msg);
+        G1CP_TestsuiteStatusPassed = FALSE;
+    };
+
+    // Return the variable's symbol pointer
+    var int symbPtr; symbPtr = MEM_GetSymbol(name);
+    return MEM_ReadInt(symbPtr + zCParSymbol_content_offset);
+};
+
+/*
+ * Check if integer variable exists
+ */
+func void G1CP_Testsuite_CheckIntVar(var string name, var int arrIdx) {
+    // Check if variable exists
+    if (!G1CP_IsIntVar(name, arrIdx)) {
+        var string msg; msg = ConcatStrings("Integer variable '", name);
+        msg = ConcatStrings(msg, "' not found/valid");
```

Comment:
Maybe we can go back to "not found" and instead of  "not found/valid". It should suffice that the kind of symbol was checked in the beginning of the message, e.g. "Integer variable".